### PR TITLE
Fix wording and punctuation in background-position doc

### PR DESCRIPTION
**Pull request by dalps**
### Description

Change "With" to "When"; improve sentence flow in the "Order" bullet point of the 2-value syntax description.

### Motivation

Readability

### Additional details

None

### Related issues and pull requests

None

### CODE REVIEW

File: files/en-us/web/css/background-position/index.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ The `background-position` property is specified as one or more `<position>` valu
...
-    - Order: When pairing keywords, placement is not important as the browser can reorder it; the value `top left` or `left top` will yield the same result. With pairing {{cssxref("&lt;length&gt;")}} or {{cssxref("&lt;percentage&gt;")}} with a keyword, the placement is important, the value defining X should come first followed by Y, the value `right 20px` is not the same as `20px right` the former is valid but the latter is invalid. The value `left 20%` or `20% bottom` is valid as X and Y values are clearly defined and the placement is right.
+    - Order: When pairing keywords, placement is not important as the browser can reorder it; the value `top left` or `left top` will yield the same result. When pairing {{cssxref("&lt;length&gt;")}} or {{cssxref("&lt;percentage&gt;")}} with a keyword, the placement is important: the value defining X should come first followed by Y, so e.g. the value `right 20px` is valid while `20px right` is not. The value `left 20%` or `20% bottom` is valid as X and Y values are clearly defined and the placement is right.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/background-position/index.md:70]:
```suggestion
    - Order: When pairing keywords, placement is not important as the browser can reorder it; the value `top left` or `left top` will yield the same result. When pairing {{cssxref("&lt;length&gt;")}} or {{cssxref("&lt;percentage&gt;")}} with a keyword, the placement is important: the value defining X should come first followed by Y, so for example the value `right 20px` is valid while `20px right` is invalid. The values `left 20%` and `20% bottom` are valid as X and Y values are clearly defined and the placement is correct.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks so much for your contribution to MDN, @dalps!

While your changes look correct, I've made a suggestion that offers a small improvement and also fixes some bad grammar in the following sentence. Does this look OK to you?

File: files/en-us/web/css/background-position/index.md (hunk 1 above)
Comment (dalps) [reply files/en-us/web/css/background-position/index.md:70]: Looks OK to me! For the sake of consistency, I would also change

> the value `top left` or `left top` will yield the same result.

to

> the _values_ `top left` _and_ `left top` will yield the same result.

in the previous sentence.

**Review by chrisdavidmills — APPROVED** [review]
Looking good; let's get this merged. Thanks again @dalps!

**Pull request merged**